### ISSUE TITLE
false positive "PHP_OS constant should be used instead."

### ISSUE THREAD

**Issue by SpacePossum** [kalessil/phpinspectionsea#1171]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Positive
| Plugin         | Php Inspections (EA Extended) v3.0.10 |
| Language level | PHP 7.0

Current behaviour (description/screenshot:)
--

```php
$simpleOS = @php_uname('s');  // PHP_OS constant should be used instead.
$os = php_uname(); // PHP_OS constant should be used instead.
```

Expected behaviour
--

No warnings

Environment details
---
PHPStorm 2018.3.3

```
$ php -v
PHP 7.2.13-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Dec  7 2018 08:07:08) ( NTS )
```

I think the warning is not correct because:

```
PHP_OS (string)
The operating system PHP was built for.
```
http://php.net/manual/en/reserved.constants.php

```
php_uname
Returns information about the operating system PHP is running on
```
http://php.net/manual/en/function.php-uname.php

Even if the value is typically close (one usually doesn't run PHP versions build on a different OS) these are not the same. Secondly when using `php_uname` passing flags the output is almost never the same as the constant.

**Comment by kalessil** [comment]
Awesome, thank you for reporting @SpacePossum !

**Comment by kalessil** [comment]
Fixed!

**Comment by SpacePossum** [comment]
thanks : D
